packages/grafana-ui/src/components/Tabs/Tabs.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { NavModelItem } from '@grafana/data';
+import { action } from '@storybook/addon-actions';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { Tabs } from './Tabs';
+import mdx from './Tabs.mdx';
+import { UseState } from '../../utils/storybook/UseState';
+
+export default {
+  title: 'UI/Tabs',
+  component: Tabs,
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+const navModelItem: NavModelItem = {
+  text: 'Mother tab',
+  children: [{ text: '1st child', active: true }, { text: '2nd child' }, { text: '3rd child' }],
+};
+
+export const Simple = () => {
+  return (
+    <UseState initialState={{ navModelItem }}>
+      {(state, onChangeTab) => {
+        return (
+          <Tabs
+            main={state.navModelItem}
```

Comment:
The naming `main` here is a bit confusing for me. I guess it might make sense in the context of navigation but if the goal is to reuse this component then I guess we probably can find a better name.

Also an idea is to only pass the children instead of the whole `NavModelItem` since we are only using those.